packages/devtools/lib/src/framework/framework.dart

```diff
@@ -136,6 +135,25 @@ class Framework {
     }
   }
 
+  void _importTimeline(Map<String, dynamic> import) {
+    TimelineScreen timelineScreen = screens.firstWhere(
+      (screen) => screen.id == timelineScreenId,
+      orElse: () => null,
+    );
+    if (timelineScreen == null) {
+      addScreen(timelineScreen = TimelineScreen(disabled: false));
+    }
+    _enterImportMode();
+    navigateTo(timelineScreenId);
+    timelineScreen.loadFromImport(import);
+  }
+
+  void _enterImportMode() {
+    connectDialog.hide();
```

Comment:
we should have a single helper that hides the connectDialog and the import message. I think we are doing that multiple places now.

Reply:
This is the only place both are called now.